engine/installation/linux/oracle.md

```diff
@@ -1,204 +1,236 @@
+    ```
 
-        $ sudo docker run hello-world
+2.  **Optional**: Enable the **testing** repository. This repository is included
+    in the `docker.repo` file above but is disabled by default. You can enable
+    it alongside the stable repository. Do not use unstable repositories on
+    on production systems or for non-testing workloads.
 
-## Optional configurations
+    > **Warning**: If you have both stable and unstable repositories enabled,
+    > installing or updating without specifying a version in the `yum install`
+    > or `yum update` command will always install the highest possible version,
+    > which will almost certainly be an unstable one.
 
-This section contains optional procedures for configuring your Oracle Linux to work
-better with Docker.
+    ```bash
+    $ sudo yum-config-manager --set-enabled docker-testing
+    ```
 
-* [Create a docker group](#create-a-docker-group)
-* [Configure Docker to start on boot](#configure-docker-to-start-on-boot)
-* [Use the btrfs storage engine](#use-the-btrfs-storage-engine)
+    You can disable the `testing` repository by running the `yum-config-manager`
+    command with the `--set-disabled` flag. To re-enable it, use the
+    `--set-enabled` flag. The following command disables the `testing`
+    repository.
 
-### Create a Docker group		
+    ```bash
+    $ sudo yum-config-manager --set-disabled docker-testing
+    ```
 
-The `docker` daemon binds to a Unix socket instead of a TCP port. By default
-that Unix socket is owned by the user `root` and other users can access it with
-`sudo`. For this reason, `docker` daemon always runs as the `root` user.
+#### Install Docker
 
-To avoid having to use `sudo` when you use the `docker` command, create a Unix
-group called `docker` and add users to it. When the `docker` daemon starts, it
-makes the ownership of the Unix socket read/writable by the `docker` group.
+1.  Update the `yum` package index.
 
->**Warning**: The `docker` group is equivalent to the `root` user; For details
->on how this impacts security in your system, see [*Docker Daemon Attack
->Surface*](../../security/security.md#docker-daemon-attack-surface) for details.
+    ```bash
+    $ sudo yum -y check-update
+    ```
+2.  Install the latest version of Docker, or go to the next step to install a
+    specific version.
 
-To create the `docker` group and add your user:
+    ```bash
+    $ sudo yum -y install docker-engine
+    ```
 
-1. Log into Oracle Linux as a user with `sudo` privileges.
+    > **Warning**: If you have both stable and unstable repositories enabled,
+    > installing or updating Docker without specifying a version in the
+    > `yum install` or `yum upgrade` command will always install the highest
+    > available version, which will almost certainly be an unstable one.
 
-2. Create the `docker` group.
+3.  On production systems, you should install a specific version of Docker
```

Comment:
You do not explain how to determine which specific version of Docker should be installed. How is the end-user supposed to determine which version of Docker to install here? Also, why shouldn't the user use the latest version of Docker? That's not explained either. Not using the latest version could leave the user vulnerable to security flaws.